packages/components/src/components/select-native/select-native.jsx

```diff
@@ -15,9 +15,25 @@ const getDisplayText = (list_items, value) => {
     return '';
 };
 
-const SelectNativeOptions = ({ list_items, should_hide_disabled_options, use_text }) => {
+const SelectNativeOptions = ({ label, list_items, should_hide_disabled_options, use_text }) => {
     const options = should_hide_disabled_options ? list_items.filter(opt => !opt.disabled) : list_items;
-
+    const grouped = options.reduce((r, a) => {
+        r[a.group] = [...(r[a.group] || []), a];
+        return r;
+    }, {});
+    const keys = Object.keys(grouped);
+    const values = Object.values(grouped);
+    if (label === 'Asset') {
```

Comment:
The component should be independent of the platform packages. I don't think it's good to check the label prop as it will be different when it is translated.

Reply:
Got it. Can you give an advice what can I use as alternate for it?

Reply:
I used your advice, thanks.